[2024-12-20T10:39:21.915+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-20T10:39:21.941+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sql_operator_dag.SQL_provider_task scheduled__2024-01-19T20:00:00+00:00 [queued]>
[2024-12-20T10:39:21.952+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sql_operator_dag.SQL_provider_task scheduled__2024-01-19T20:00:00+00:00 [queued]>
[2024-12-20T10:39:21.954+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-20T10:39:22.032+0000] {taskinstance.py:2889} INFO - Executing <Task(SQLExecuteQueryOperator): SQL_provider_task> on 2024-01-19 20:00:00+00:00
[2024-12-20T10:39:22.045+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'sql_operator_dag', 'SQL_provider_task', 'scheduled__2024-01-19T20:00:00+00:00', '--job-id', '5401', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgresql.py', '--cfg-path', '/tmp/tmpqlke6iwv']
[2024-12-20T10:39:22.050+0000] {standard_task_runner.py:105} INFO - Job 5401: Subtask SQL_provider_task
[2024-12-20T10:39:22.047+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=15625) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-20T10:39:22.066+0000] {standard_task_runner.py:72} INFO - Started process 15632 to run task
[2024-12-20T10:39:22.150+0000] {task_command.py:467} INFO - Running <TaskInstance: sql_operator_dag.SQL_provider_task scheduled__2024-01-19T20:00:00+00:00 [running]> on host a998b5b3d359
[2024-12-20T10:39:22.299+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mateo' AIRFLOW_CTX_DAG_ID='sql_operator_dag' AIRFLOW_CTX_TASK_ID='SQL_provider_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-19T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-19T20:00:00+00:00'
[2024-12-20T10:39:22.302+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-20T10:39:22.367+0000] {sql.py:278} INFO - Executing: 
        create table if not exists dag_runs_v2(
            dt date,
            dag_id varchar,
            primary key (dt,dag_id) 
        )
        
[2024-12-20T10:39:22.381+0000] {base.py:84} INFO - Retrieving connection 'sql_operator_practice'
[2024-12-20T10:39:22.405+0000] {base.py:84} INFO - Retrieving connection 'sql_operator_practice'
[2024-12-20T10:39:22.460+0000] {sql.py:553} INFO - Running statement: 
        create table if not exists dag_runs_v2(
            dt date,
            dag_id varchar,
            primary key (dt,dag_id) 
        )
        , parameters: None
[2024-12-20T10:39:22.482+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-20T10:39:22.484+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=sql_operator_dag, task_id=SQL_provider_task, run_id=scheduled__2024-01-19T20:00:00+00:00, execution_date=20240119T200000, start_date=20241220T103921, end_date=20241220T103922
[2024-12-20T10:39:22.587+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-20T10:39:22.626+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-20T10:39:22.632+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
